Tests/LibWeb/Text/input/css/animate-with-many-keyframes.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<style>
+    #foo {
+        animation: 1s forwards linear anim;
+        animation-play-state: paused;
+        position: absolute;
+    }
+
+    @keyframes anim {
+        0% {
+            left: 0px;
+            top: 0px;
+        }
+        50% {
+            left: 100px;
+            top: 0px;
+        }
+        100% {
+            left: 100px;
+            top: 100px;
+        }
+    }
+</style>
+<div id="foo"></div>
+<script src="../include.js"></script>
+<script>
+    asyncTest(async (done) => {
+        const foo = document.getElementById("foo");
+        const anim = foo.getAnimations()[0];
+        anim.onfinish = function () {
+            const rect = foo.getBoundingClientRect();
+            println("rect.left > 0: " + (rect.left > 0));
+            println("rect.left <= 100: " + (rect.left <= 100));
+            println("rect.top > 0: " + (rect.top > 0));
```

Comment:
Would it work to just print the rect's left/top/right/width in one go? That'd protect against other animation regressions related to the last keyframe as well.

Reply:
I tried that at first but found it was flake-prone because the `top` offset seems to end at a slightly different position on each run (e.g. 99.921875 then 99.90625 on consecutive runs)

Reply:
Oh let's leave this as-is then. Definitely sounds like a bug!